src/megaclient.cpp

```diff
@@ -4611,7 +4536,7 @@ bool MegaClient::procsc()
                             //TODO: remove android control after android gives green light to this.
                             enabletransferresumption();
 #endif
-                            resumeResumableSyncs();
+                            syncs.resumeResumableSyncsOnStartup();
```

Comment:
This call results in a segmentation fault upon login into a folder link. It attempts to access to `Syncs::mSyncConfigDb` and it's invalid. Folder links cannot be synced (no permission for uploads), so my suggestion is to protect the code with a `if (mClient->loggedinfolderlink()) return;` inside the method.

Reply:
Thanks - updated with exactly that change

Reply:
Actually, how did any syncs get loaded if it's not the same account that saved them?   Surely the file/db should be encrypted to this account to prevent accidents like that.

Reply:
Well, I think the issue is that logging into a folder link, the `me` handle is the public handle of the folder link. The db is named like: `megaclient_statecache<version>_syncconfigs_<user_handle>.db`, so it attempts to find a file that doesn't exist, so `mSyncConfigDb` is invalid.
FYI, the records are encrypted to the user's Master Key ;)